### DISCUSSION TITLE
Grafix purpose?!

### DISCUSSION THREAD

**Discussion by KitOdin** [FunkinCrew/Funkin#1693] [Q&A] [answered]
Ayo my right hand man ninjamuffin99, whats the purpose of grafix, and why was it never used.
i need ANSWERS!

**Comment by Cheemsandfriends** [comment]
Because it was like a thing to collect all stuff in an alpha of the game I guess.
Then They separated everything and redrawed the icons.
(maybe lol)

**Comment by steviegt6** [comment, marked as answer]
It appears that it once held some assets that are no longer used. There's some assets in there that have been moved else-where, what seems like a concept for BF's icon featuring two extra "poses", and a Daddy Dearest icon with an actual losing face. Likely just a remnant from early code (perhaps the Ludum Dare prototype?).

**Reply by steviegt6** [reply]
Alright, hmm, definitely appears it's left-over old stuff. Looks like it held images for some basic assets, and they were also cut up individually?
Original commit added them: 6568703a7d18f59d6fdc7fb93c64e52b71e11693
Commit that comes immediately after, using the independent sprites instead of the collection in grafix: a380fa16e5ad521fde48fce10073be771f9ab347
